contrib/python/tests/python/pants_test/contrib/python/checks/tasks/checkstyle/test_except_statements.py

```diff
@@ -25,12 +26,15 @@ def test_except_statements(self):
     for clause in ('except:', 'except :', 'except\t:'):
       self.assertNit(EXCEPT_TEMPLATE.format(clause), 'T803')
 
-    for clause in (
-        'except KeyError, e:',
-        'except (KeyError, ValueError), e:',
-        'except KeyError, e :',
-        'except (KeyError, ValueError), e\t:'):
-      self.assertNit(EXCEPT_TEMPLATE.format(clause), 'T601')
+    try:
+      for clause in (
+          'except KeyError, e:',
+          'except (KeyError, ValueError), e:',
+          'except KeyError, e :',
+          'except (KeyError, ValueError), e\t:'):
+        self.assertNit(EXCEPT_TEMPLATE.format(clause), 'T601')
+    except CheckSyntaxError:  # Fix Python 3 raising SyntaxError
```

Comment:
See above discussion about print statements raising syntax error. Feeding the above input into `assertNit` raises a `SyntaxError`.